dev/tasks/python-wheels/osx-build.sh

```diff
@@ -194,9 +194,8 @@ function install_wheel {
     pushd $1  # enter arrow's directory
     wheelhouse="$PWD/python/dist"
 
-    # Install compatible wheel
-    pip install $(pip_opts) \
-        $(python $multibuild_dir/supported_wheels.py $wheelhouse/*.whl)
+    # Install wheel
+    pip install $(pip_opts) $wheelhouse/*.whl
```

Comment:
Something must have changed with the `wheel` package version, [supported_wheels.py](https://github.com/matthew-brett/multibuild/blob/devel/supported_wheels.py) was also updated lately.

We build single wheel per travis job, so filtering is not required and `pip install` should fail if the wheel is not suitable for the platform.